Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian:latest
+
+RUN \
+apt-get update -y && \
+apt-get install default-jre -y
+
+ADD ./build/libs/tsunami-security-scanner-utilities.jar tsunami-test-hello-world.jar
```

Comment:
This requires us to build the jar before calling `docker build`.  Ideally we could use a Dockerfile with [a multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds). But this doesn't necessarily block this PR.